docs/topics/auth/default.txt

```diff
@@ -1292,6 +1292,10 @@ implementation details see :ref:`using-the-views`.
 
     Sample ``registration/password_reset_email.html`` (email body template):
 
+    .. versionadded:: 1.9
+
+    Passes ``extra_email_context`` parameter to the opt dictionary used for email template
```

Comment:
Please add the new parameter to the `password_reset()` signature above.
missing period for this sentence.
Please wrap documentation at 79 chars.